src/pynws/raw_data.py

```diff
@@ -20,11 +20,37 @@ def get_header(userid: str) -> Dict[str, str]:
     return {"accept": API_ACCEPT, "User-Agent": API_USER.format(userid)}
 
 
+def get_params(
+    limit: Optional[int] = 0,
+    start: Optional[datetime] = None,
+    units: Optional[str] = None,
```

Comment:
This function now mixes multiple params from multiple endpoints.  Could this result in problems if we have conflicting params with the same name in different endpoints?

Reply:
Yes, it could. I can write a "forecast_parms()" function instead, for the functions that need it, and move the other ones back in the observations.

Reply:
I am back tracking here, but this is a purely internal helper function.  I think this function is fine as is for now as it only has three items in it.  We can always break it up later into separate functions if it becomes too unwieldy.